index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Messi Fever!</title>
+
+		<link rel="stylesheet" type="text/css" href="style.css">
+ 	</head>
+	
+	<body>
+		
+		<nav>
+			<a href="index.html"> Home </a> |
+			<a href="experience/experience.html" target="_blank"> Resume and Honors</a> |
+			<a href="http://www.fcbarcelona.com/all-about-leo-messi" target="_blank"> Official Messi site</a> |
+			<a href="http://www.worldsoccershop.com/shop-by-player-messi--lionel.html" target="_blank"> Buy Gear </a>
+		</nav>
+
+		<h1>Messi Fever! </h1>
+		<img src="lionel messi cover.jpeg" text="Lionel Messi" class="main" >
+		<h3>La Pulga "The Flea"</h3>
+		
+
+		<hr/>
+		<h3 class="flea">
+			What's that? The Flea? 
+			</h3>
+		
+ 		<p>
+			Yeah, Messi's nickname is "La Pulga," and that's Spanish for "the flea."
+
+			To us, it doesn't get much better than that. He's little. He harasses defenders. He never goes away and gets under your skin. It fits. <br/>
+
+			And anyway, if you can turn a wimpy-sounding nickname like "The Flea" into a descriptor of the 
+			<a href="https://www.genius.com/Lionel-messi-messis-career-honours-annotated" target="_blank"> best player in the world </a> , you've clearly done something right.
+			
+		</p>
+
+			<img class="action" src="in action messi.jpg" text="Lionel Messi" >
```

Comment:
:hammer: This image won't show like this. This is a good example of why we want to name out files with either underscores (_) or dashes (-) in them. The file you have is `in action Messi.jpg` but you can't use spaces in attributes. I would rename the file to `in_action_messi.jpg`. IF you wanted to use it the way it is, then you need to represent spaces with a `%20` like this: `in%20action%20messi.jpg`. This gets messy though and is a pain to keep doing.